.github/ISSUE_TEMPLATE/4-question.yml

```diff
@@ -0,0 +1,10 @@
+name: ❓ Need help
```

Comment:
This is probably better handled with Gitter and the Discussions feature than an Issue

Reply:
Yes, ideally to be moved.